index.php

```diff
@@ -4,7 +4,7 @@
 
 <html lang="en">
 <!DOCTYPE html>
-<html>
+<body>
```

Comment:
So, this still isn't valid html. It needs to look like this:
```
<!DOCTYPE html>
<html lang="en">
    <head>
        <!-- Your link tags for your styles and a title tag -->
    </head>
    <body>
        <!-- Your contents: divs, h* tags, nav, footer, inputs... everything else -->
    </body>
</html>
```
Let me (or any of the other trainers ofc) know if you want to step into a breakout room to get it worked out